tools/trap/cli/logParser.py

```diff
@@ -574,6 +578,90 @@ def print_wrong_sp(self):
 								stack_val = stack_val - int(self.g_stext_app[is_app_symbol - 1])
 							utils.print_symbol(stack_addr, stack_val, is_app_symbol, self.bin_path, self.app_name)
 
+	# API to parse asserted tcb information
+	def parse_tcb_info(self):
+		TCB_FLAG_TTYPE_SHIFT      = (0)	                         # Bits 0-1: thread type 
```

Comment:
Do we need to declare constants local to function?
Can we move them inside class declarations?

Reply:
Okay.. It's better to declare them global rather confine them to just class?